pages/hooks.mdx

```diff
@@ -0,0 +1,19 @@
+import nativeEvent from '../examples/files/react/nativeEvent.js'
+
+# Hooks
+
+We've already had a brief introduction to Hooks in Chapter 3.5 with Events, but now let's examine it more carefully. 
+
+Hooks refer to a set of library functions provided by React that allows the developer to handle UI state in a cleaner and more functional way. 
+
+We can define state as set of data in the application that is mutable over time (or by user interaction) and is kept within the confines of a React component. 
```

Comment:
I was thinking in a general sense! I don't think there's been an introduction to what "UI state" is in a React application thus far in the tutorial, so I was thinking of defining it. What do you think?

Reply:
Hmm... I think this is helpful. I might limit the scope of the definition a bit, since state can have so many meanings e.g. "When we talk about "state" in a React app, we're usually referring to the set of data..."